<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form Data Buku</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>541</width>
     <height>451</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabDetailBuku">
    <attribute name="title">
     <string>Data Buku Detail</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Judul Buku</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtJudulBukuDetail">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableDetailBuku">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>120</y>
       <width>551</width>
       <height>301</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Judul Buku</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Total Halaman</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Harga</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Stock</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="btnSearch">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>10</y>
       <width>91</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Total Halaman</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Harga</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtTTLHal">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtHargaDetail">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabEditBuku">
    <attribute name="title">
     <string>Edit Data Buku</string>
    </attribute>
    <widget class="QTableWidget" name="tableEditBuku">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>541</width>
       <height>381</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAdd">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>390</y>
       <width>91</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Tambah</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>390</y>
       <width>91</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Ubah</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDelete">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>390</y>
       <width>91</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Hapus</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
